.github/workflows/main.yml

```diff
@@ -29,60 +31,58 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-14]
+        os: [ubuntu-latest, windows-latest, macos-latest]
     steps:
-      - name: Set up repository
-        uses: tree-sitter/parser-setup-action@v1.2
-        with:
-          node-version: 20
+      - name: Clone repository
```

Comment:
nitpick/personal preference/not a blocker: I personally tend to dislike adding `name` to common actions because it's usually self-explanatory what the action does. It gives off

```
for i in range(4) # i is an iterator
```

comment vibes.